### PR TITLE
7740 - feature flag cleanup

### PR DESCRIPTION
**Pull request by jongilbraith**
Went through all feature toggles that are present in the codebase and where possible removed them, removing any toggled out code where appropriate.  In the case of the 'feedback' toggle (different from the recent 'page feedback' feature) there were quite a few related bits that came out.

Additionally, one toggle controlled a feature flag in the budget planner which has been removed in [this PR](https://github.com/moneyadviceservice/budget_planner/pull/471). The WIP commit updating the Gemfile to point to the feature branch will be rebased out when it's been merged and a new version of the gem has been published.

Along with this is [a PR](https://github.com/moneyadviceservice/scripts/pull/837) for scripts which updates all the environments'  features.yml templates.

---
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1608)

### CODE REVIEW

**Comment by jriga** [comment]
great job 👍

**Comment by rnalexander** [comment]
Looks good!  Also good job of separating concerns in the commits!  It really helped me review it.  (got a bit confused when I saw some internationalization go away but I figured it out.  :+1: :shipit:

**Comment by jongilbraith** [comment]
@jriga @rnalexander thanks both

File: app/views/styleguide/_navigation.html.erb [hunk 1]
```diff
@@ -54,7 +54,7 @@
           <li><%= link_to 'Child category page', pages_child_category_page_styleguide_path %></li>
           <li><%= link_to 'Error page', pages_error_styleguide_path %></li>
           <li><%= link_to 'Tool', pages_tool_styleguide_path %></li>
-          <li><%= link_to 'Feedback - technical', pages_feedback_technical_styleguide_path %></li>
+          <li><%= link_to 'Technical feedbak', pages_technical_feedback_styleguide_path %></li>
```
Comment (norm) [inline app/views/styleguide/_navigation.html.erb]: "Feedback"

Comment (jongilbraith) [inline app/views/styleguide/_navigation.html.erb]: Nice catch 👀

**Pull request merged**